### ISSUE TITLE
2 last layers have softmax activation

### ISSUE THREAD

**Issue by grig-guz** [ubclaunchpad/minutes#133]
https://github.com/ubclaunchpad/minutes/blob/b13d33b83b341aab96198feb300ea5d010f982f2/minutes/minutes.py#L55

Second-to-last layer should have a usual activation function, not a loss function.
